gapis/api/vulkan/transform_command_splitter.go

```diff
@@ -907,6 +907,11 @@ func (splitTransform *commandSplitter) Split(ctx context.Context, id api.SubCmdI
 	return nil
 }
 
+// Remove removes a draw call command from a command buffer.
+func (splitTransform *commandSplitter) Remove(ctx context.Context, id api.SubCmdIdx) error {
```

Comment:
To keep a consistent naming, should this be `Disable`? (Or should the flag be `removeCmd` ?)

Reply:
Disabling is the name of the feature and remove is the process that needed for that. 

Disclaimer: The function name may change in the future with the details of implementation comes up.